pkg/utils/utils.go

```diff
@@ -62,11 +61,11 @@ func GetLastModifiedTime(dbname, pname string) string {
 
 	switch fmt.Sprint(lastModTime.Date()) {
 	case fmt.Sprint(today.Date()):
-		return fmt.Sprintf("Today %s", lastModTime.Format("15:04"))
+		return fmt.Sprintf("Today %s", lastModTime.Format("15:04")), int64(lastModTime.Unix())
 	case fmt.Sprint(today.AddDate(0, 0, -1).Date()):
-		return fmt.Sprintf("Yesterday %s", lastModTime.Format("17:00"))
+		return fmt.Sprintf("Yesterday %s", lastModTime.Format("17:00")), int64(lastModTime.Unix())
 	}
-	return fmt.Sprint(lastModTime.Format("02 Jan 06 15:04"))
+	return fmt.Sprint(lastModTime.Format("02 Jan 06 15:04")), int64(lastModTime.Unix())
```

Comment:
This approach is strange. You should returned a date, and let the caller transform it